js/ui/overlay/ui.overlay.js

```diff
@@ -102,15 +101,7 @@ const Overlay = Widget.inherit({
                 return;
             }
 
-            e.preventDefault();
-            e.stopPropagation();
-
-            const allowedOffsets = this._allowedOffsets();
-            const offset = {
-                top: fitIntoRange(top, -allowedOffsets.top, allowedOffsets.bottom),
-                left: fitIntoRange(left, -allowedOffsets.left, allowedOffsets.right)
-            };
-            this._changePosition(offset);
+            this._draggable.moveTo(top, left, e);
```

Comment:
maybe `_drag` instead of `_draggeble` to emphasize attention that we do not use Draggable component?

Reply:
ok